### ISSUE TITLE
Playground is broken with Math

### ISSUE THREAD

**Issue by Zamiell** [TypeScriptToLua/TypeScriptToLua.github.io#117]
https://typescripttolua.github.io/play#code/MYewdgzgLgBAZiEMC8MCyBDKALAdHAG0QCcAKARl3IEoBuAKCA

For reference, the real TSTL will transpile it to:

```lua
--[[ Generated with https://github.com/TypeScriptToLua/TypeScriptToLua ]]
foo = math.floor(1.1)
```

So it just seems to be a playground problem.

**Comment by Zamiell** [comment]
Also this is a recent regression, as I'm 99% sure that this works fine a few months ago.

**Comment by Perryvw** [comment]
Not sure what changed, but TS was not detecting TS lib files as standard library files due to the default filename monaco was making up for the lib files as we added them. I explicitly added the file name now, and the files are detected as standard library again.
